config/config.go

```diff
@@ -1217,6 +1217,7 @@ func (c *Config) missingTomlField(_ reflect.Type, key string) error {
 
 	// Serializer options to ignore
 	case "prefix", "template", "templates",
+		"avro_schema", "avro_schema_registry", "avro_measurement", "avro_tags", "avro_fields", "avro_timestamp", "avro_timestamp_format", "avro_field_separator",
```

Comment:
Can you please check if the parser works without setting those as they are scheduled for removal anyway. Furthermore, serializers would be the wrong place anyway. ;-)